src/core.ts

```diff
@@ -34,8 +34,8 @@ export class Eta {
   renderStringAsync = renderStringAsync;
 
   filepathCache: Record<string, string> = {};
-  templatesSync = new Cacher<TemplateFunction>({});
-  templatesAsync = new Cacher<TemplateFunction>({});
+  templatesSync: Cacher<TemplateFunction> = new Cacher<TemplateFunction>({});
```

Comment:
These annotations are useless thanks to TypeScript type inference.

Reply:
Yep, JSR complains though; it won't generate docs for types that take too many cycles to resolve; running the dry-run it said to add explicit types for those interfaces to enable auto-docs.

Reply:
OK, now I understand 😄

Reply:
Any speedup is probably imperceptible on these in any non-constrained system, but I figured it wouldn't hurt to add them.